src/test/java/io/phasetwo/service/mapper/OrganizationSpecificAttributeMapperTest.java

```diff
@@ -0,0 +1,97 @@
+package io.phasetwo.service.mapper;
+
+import static io.phasetwo.service.Helpers.createUserWithCredentials;
+import static io.phasetwo.service.Helpers.deleteUser;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import io.phasetwo.client.openapi.model.OrganizationRepresentation;
+import io.phasetwo.service.AbstractOrganizationTest;
+import io.phasetwo.service.protocol.oidc.mappers.OrganizationSpecificAttributeMapper;
+import io.restassured.response.Response;
+import jakarta.ws.rs.core.Response.Status;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.extern.jbosslog.JBossLog;
+import org.junit.jupiter.api.Test;
+import org.keycloak.TokenVerifier;
+import org.keycloak.admin.client.resource.RealmResource;
+import org.keycloak.protocol.oidc.OIDCLoginProtocol;
+import org.keycloak.protocol.oidc.mappers.OIDCAttributeMapperHelper;
+import org.keycloak.representations.AccessToken;
+import org.keycloak.representations.idm.ClientRepresentation;
+import org.keycloak.representations.idm.ProtocolMapperRepresentation;
+import org.keycloak.representations.idm.UserRepresentation;
+
+@JBossLog
+class OrganizationSpecificAttributeMapperTest extends AbstractOrganizationTest {
+
+  public static final String CLAIM = "secret_attr";
+
+  @Test
+  void shouldConfigureOrganizationSpecificAttributeMapperOidcProtocolMapper() throws Exception {
+    // add organization with attribute 'secret_attr'
+    var organization = createOrganization(
```

Comment:
Could we add a second organization to the set of user organizations? In the second organization I would not add the CLAIM to test what is the behaviour if the claim is not available.

Reply:
Added 4 organization and added one extra claim mapper for the validation.
- Organization 1 with secret_attr value My Secret Value
- Organization 2 with secret_attr value My Second Secret Value
- Organization 3 with secret_attr value empty string
- Organization 4 without secret_attr value and another attribute another_attribute with value  My Value